src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -47,6 +49,7 @@ public List<ConversionUnit> SelectedToConversionSource
             }
         }
 
+        [JsonProperty("MetricConversion")]
```

Comment:
The schema name is MetricValues.  It can be changed if we prefer MetricConversion.  (I do prefer MetricConversion, MetricValues came from Flood.)